client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+
+class SurveyDifficultyContainer extends Component {
+  static navigationOptions = {
+    title: "LESSONS"
+  };
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
+        <Text
+          style={{
+            paddingBottom: 40,
+            textAlign: "center",
+            fontSize: 18
+          }}
+        >
+          What's your level
+          {"\n"}
+          of financial knowledge?
+        </Text>
+
+        <View
+          style={{
```

Comment:
Put it into styles file

Reply:
I didn't rebase, it is already in develop